test/jdk/java/awt/event/MouseEvent/SpuriousExitEnter/ResizeMouseExitEnterMisfire.java

```diff
@@ -0,0 +1,110 @@
+import java.awt.Rectangle;
```

Comment:
Any code we submit to the openjdk workspace should have a proper copyright header - test included. You can borrow it from any recently submitted or edited test.